src/global.d.ts

```diff
@@ -0,0 +1,11 @@
+export {};
+
+declare global {
+  /**
+   * Now declare things that go in the global namespace,
+   * or augment existing declarations in the global namespace.
+   */
+  interface Insights {
+    chrome: any;
+  }
```

Comment:
If we do this, do we need to declare this in `App.tsx`, too, or can we make this suffice?